test/Microsoft.AspNetCore.Server.KestrelTests/MemoryPoolIteratorTests.cs

```diff
@@ -1,1449 +1,1449 @@
-                for (var length = vectorSpan / 2; length <= vectorSpan * 3 + vectorSpan / 2; length += vectorSpan / 2)
-                {
-                    // ...place the seek char at vector and input boundaries...
-                    for (var i = Math.Min(vectorSpan - 1, length - 1); i < length; i += ((i + 1) % vectorSpan == 0) ? 1 : Math.Min(i + (vectorSpan - 1), length - 1))
-                    {
-                        var input = new StringBuilder(new string('a', length));
-                        input[i] = 'b';
-
-                        // ...along with sentinel characters to seek the limit iterator to...
-                        input[i - 1] = 'A';
-                        if (i < length - 1) input[i + 1] = 'B';
-
-                        // ...and check that Seek() succeeds with a limit iterator at or past the seek char position...
-                        data.Add(new object[] { input.ToString(), 'b', 'b', 'b' });
-                        if (i < length - 1) data.Add(new object[] { input.ToString(), 'b', 'B', 'b' });
-
-                        // ...and fails with a limit iterator before the seek char position.
-                        data.Add(new object[] { input.ToString(), 'b', 'A', -1 });
-                    }
-                }
-
-                return data;
-            }
-        }
-    }
-}
\ No newline at end of file
+//// Copyright (c) .NET Foundation. All rights reserved.
+//// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+//using System;
+//using System.Collections.Generic;
+//using System.Linq;
+//using System.Numerics;
+//using System.Text;
+//using Microsoft.AspNetCore.Server.Kestrel.Internal.Infrastructure;
+//using Xunit;
+
+//namespace Microsoft.AspNetCore.Server.KestrelTests
+//{
+//    public class MemoryPoolIteratorTests : IDisposable
+//    {
+//        private readonly MemoryPool _pool;
+
+//        public MemoryPoolIteratorTests()
+//        {
+//            _pool = new MemoryPool();
+//        }
+
+//        public void Dispose()
+//        {
+//            _pool.Dispose();
+//        }
+
+//        [Fact]
+//        public void TestFindFirstEqualByte()
+//        {
+//            var bytes = Enumerable.Repeat<byte>(0xff, Vector<byte>.Count).ToArray();
+//            for (int i = 0; i < Vector<byte>.Count; i++)
+//            {
+//                Vector<byte> vector = new Vector<byte>(bytes);
+//                Assert.Equal(i, MemoryPoolIterator.LocateFirstFoundByte(vector));
+//                bytes[i] = 0;
+//            }
+
+//            for (int i = 0; i < Vector<byte>.Count; i++)
+//            {
+//                bytes[i] = 1;
+//                Vector<byte> vector = new Vector<byte>(bytes);
+//                Assert.Equal(i, MemoryPoolIterator.LocateFirstFoundByte(vector));
+//                bytes[i] = 0;
+//            }
+//        }
```

Comment:
Before merging can we move whatever tests belong in corefxlabs there? Keep whatever tests still work with the "MemoryPoolIteratorExtensions"? And delete whatever no longer makes sense?